### PR TITLE
New snmptrap transport

### PR DESCRIPTION
**Pull request by ciroiriarte**
New snmptrap transport. Usually required to report events to northbound platforms. Compliant with [RFC2578](https://www.rfc-editor.org/rfc/rfc2578)

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Alerting/Transports.md [hunk 1]
```diff
@@ -701,6 +701,74 @@ You need a token you can find on your personnal space.
...
+
+| Config | Example |
+| ------ | ------- |
+| Transport type | Snmptrap |
+| Destination host | noc.mycompany.com |
+| SNMP Trap transport | UDP |
+| Community | out-community |
+| Trap Definition | TTC-NOTIFICATIONS-MIB::daEvent |
+| PDU | TRAPv2 |
+| Binary path | /usr/bin/snmptrap |
```
Comment (murrant) [inline doc/Alerting/Transports.md:722]: We have tools to find binaries and net-snmp is already required by LibreNMS.

File: mibs/ttc/TTC-NOTIFICATIONS-MIB.mib [hunk 2]
```diff
@@ -0,0 +1,945 @@
+TTC-NOTIFICATIONS-MIB DEFINITIONS ::= BEGIN
```
Comment (murrant) [inline mibs/ttc/TTC-NOTIFICATIONS-MIB.mib:1]: What is this MIB?  It has references to icinga in it?

(also the file name is wrong, it should match the mib name.)

Comment (ciroiriarte) [reply mibs/ttc/TTC-NOTIFICATIONS-MIB.mib:1]: Is a MIB file created for Icinga & LibreNMS notifications. Net-snmp lookup seems to work fine with that file name. Nonetheless, will rename from TTC-NOTIFICATIONS-MIB.mib to TTC-NOTIFICATIONS-MIB as requested.

File: doc/Alerting/Transports.md (hunk 1 above)
Comment (ciroiriarte) [reply doc/Alerting/Transports.md:722]: Any pointers?

**Comment by ciroiriarte** [comment]
Will close this pull request & resubmit. Apparently I got unrelated commits in it.

**Comment by murrant** [comment]
FYI, you don't have to close to fix issues, just commit the changes to your branch and it will update the PR.

**Comment by ciroiriarte** [comment]
didn't know how to :)

had an unwanted change in the PR, fiddled with it and got a branch mess AND the unwanted change.

**Comment by murrant** [comment]
@ciroiriarte it might not be obvious because you don't actually need to do anything :) Just push commits to the branch on your fork :)

**Comment by ciroiriarte** [comment]
I messed up branches in my repository :)

Latest attempt at https://github.com/librenms/librenms/pull/14350

File: doc/Alerting/Transports.md (hunk 1 above)
Comment (ciroiriarte) [reply doc/Alerting/Transports.md:722]: Hope this is the right approach: https://github.com/librenms/librenms/commit/5a7ec3ba1d7f584d8491e1092d5337c6fb0cdf04

**Pull request closed without merging**